docs/src/modules/components/MarkdownElement.js

```diff
@@ -32,11 +32,9 @@ renderer.heading = (text, level) => {
   return [
     `<h${level}>`,
     `<a class="anchor-link" id="${escapedText}"></a>${text}`,
-    typeof window !== 'undefined'
```

Comment:
Not sure if this always issued a hydration warning or not. Either way it still does. Probably better to move it to an external image but this is fine during work.

Reply:
Will revisit in a follow-up. Added markup is negligible compared to the amount of JS we ship (which usually is proportional to the amount of headings).

Reply:
What's the % impact on the initial index.html payload?
I haven't noticed the hydration warning when introducing this diff. Oops. What about using an external image like, you did for the support markdown page?